airflow/providers/docker/operators/docker.py

```diff
@@ -90,6 +90,12 @@ class DockerOperator(BaseOperator):
     :param host_tmp_dir: Specify the location of the temporary directory on the host which will
         be mapped to tmp_dir. If not provided defaults to using the standard system temp directory.
     :param network_mode: Network mode for the container.
+        It can be one of the following:
+            bridge - Create new network stack for the container with default docker bridge network
+            None - No networking for this container
+            container:<name|id> - Use the network stack of another container specified via <name|id>
+            host - Use the host network stack. Incompatible with `port_bindings`
```

Comment:
In host network stack, `port_binding` option is incompatible as container and it's host shares same networking namespace and the container has the same IP address as that of the host.  However in `create_host_config` function in DockerOperator we don't support port_binding as param. So I am not sure if we have to add this detail `Incompatible with port_bindings` in the documentation.